h/views/admin_groups.py

```diff
@@ -86,8 +86,9 @@ def on_success(appstruct):
             else:
                 raise Exception('Unsupported group type {}'.format(type_))
 
-            # Update group memberships
-            svc.update_membership(group, appstruct['members'])
+            # Add members to the group
```

Comment:
I opted to create a separate method _on the service_ to be explicit about the difference between adding and updating users. There are a couple of other valid approaches that could have worked here:

* The view could take the trouble to add the creator into the `userid` list when creating a new group and still use `update_members`. I opted against this because it would have required some futzing with logic around when the creator _should_ be a member initially (yes for restricted groups, no for open groups).
* The `update_members` method could have taken an additional param or something that could instruct it to be additive only (non-destructive). This felt flimsy to me.

Reply:
This is fine. Looping over `member_userids` and calling `svc.member_join(userid)` would have been fine as well, albeit marginally less convenient to test.